scipy/linalg/flapack_pos_semidef.pyf.src

```diff
@@ -0,0 +1,48 @@
+! Signatures for f2py-wrappers of FORTRAN LAPACK Symmetric Positive Semidefinite Matrix functions.
+!
+
+subroutine <prefix2>pstrf(n,a,lda,piv,rank_bn,tol,work,info,lower,clean)
```

Comment:
Why only prefix2? pstrf also has real and double flavors.

Reply:
OK, done.